src/main/java/step3/service/CarsDto.java

```diff
@@ -0,0 +1,40 @@
+package step3.service;
+
+import step3.view.CarMovementResult;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class CarsDto {
+    private final List<Car> cars = new ArrayList<>();
+
+    public static CarsDto randomStrategyCars(int carCount) {
+        return new CarsDto(
+                IntStream.range(0, carCount)
+                        .mapToObj(idx -> Car.randomMovingCar())
+                        .collect(Collectors.toList())
+        );
+    }
+
+    private CarsDto(List<Car> cars) {
+        if (cars != null && cars.size() > 0) {
```

Comment:
`List`의 `isEmpty()`를 활용하면 `0`과 같은 불필요한 숫자 리터럴을 사용하지 않아도 될 것 같습니다.
```suggestion
        if (cars != null && !cars.isEmpty()) {
```

Reply:
어이쿠 무슨 생각을 하고 짠건지 참.. 하핳 수정했습니다!